app/services/pricing_service.py

```
@@ -123,6 +131,13 @@ def _parse_rules(self, raw: List[Dict[str, Any]]) -> List[PriceRule]:
         """JSONB 원본을 PriceRule 리스트로 변환"""
         return [PriceRule(**cfg) for cfg in raw]
 
+    def _is_season_active(self, season: str, target: datetime) -> bool:
```

Ruff (0.15.2)

[warning] Unused method argument: `season`

(ARG002)